Translation corrections (#442)
07th-mod/higurashi-console-arcs: Update/ura_011.txt

			NULL, "\"...Well, that's why I need your help on this particular case.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_maeno1002", 256, TRUE);
	OutputLine(NULL, "上からのストップに逆らう真似までさせて、申し訳ない限りだがな」",

> 			NULL, " I'm really sorry to make you defy the higher-ups' stop order.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I'm really sorry to make you disobey a stop order from the higher-ups.\"", Line_Normal);
```
